src/test/java/com/faforever/client/teammatchmaking/TeamMatchmakingControllerTest.java

```diff
@@ -57,18 +60,28 @@
   private FafService fafService;
   @Mock
   private EventBus eventBus;
+  @Mock
+  private Preferences preferences;
+  @Mock
+  private MatchmakerPrefs matchmakerPrefs;
```

Comment:
Rather than mocking the preferences you can just use the preferencesBuilder to create one and pass that